modules/custom/az_event/config/install/core.entity_view_display.node.az_event.az_row.yml

```diff
@@ -18,10 +18,10 @@ dependencies:
     - field.field.node.az_event.field_az_trellis_id
     - node.type.az_event
   module:
+    - az_core
     - date_ap_style
     - field_group
     - smart_title
-    - text
```

Comment:
Is this needed in the other files? It seems to work fine in them. This came up in config export from my local dev.

Reply:
Oops, yes, thanks! - it seems like this is probably necessary in the other files. I did not rexport them, just changed the text formatter.